core/templates/pages/exploration-player-page/learner-experience/input-response-pair.component.html

```diff
@@ -8,7 +8,7 @@
 </ng-template>
 
 <div class="conversation-skin-learner-answer-container">
-  <img class="conversation-skin-user-avatar rounded-circle" [src]="profilePicture" alt="">
+  <img class="conversation-skin-user-avatar rounded-circle" [src]="decodedProfilePicture ? decodedProfilePicture : ''" alt="">
```

Comment:
During the duration between injection of the template into DOM and execution of ngOnInit, decodedProfilePicture will be undefined. This can cause a similar flake with a request to backend at the `/undefined` route.